languages/elixir/exercises/concept/charlists/.docs/instructions.md

```diff
@@ -0,0 +1,41 @@
+You are working as a system administrator for a big company in Münich, Germany. One of your responsibilities is managing email accounts.
+
+You have been hearing complaints from people saying they are unable to write emails to Mr. Müller. You quickly realize that most of the company uses an old email client that doesn't recognize `müller@firma.de` as a valid email address because of the non-latin character.
+
+Telling people to give up their favorite old email client is a lost battle, so you decide to create sanitized aliases for all email accounts.
+
+## 1. Sanitize existing usernames by removing everything but lowercase letters
+
+Write a `sanitize/1` function. It should accept a username as a charlist and return the username with all characters but lowercase letters removed.
```

Comment:
Tiny nit: there are is a `sanitize/1` function defined in the stub. Should this possibly be:

```suggestion
Implement the `sanitize/1` function. It should accept a username as a charlist and return the username with all characters but lowercase letters removed.
```

?